Logix/Collections/CollectionsIterate.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using BaseX;
+using NEOSPlus;
+
+namespace FrooxEngine.LogiX.Collections
+{
+    [NodeName("Foreach")]
+    [Category("LogiX/Collections")]
+    [NodeDefaultType(typeof(CollectionsIterate<dummy, IEnumerable<dummy>>))]
+    public class CollectionsIterate<T, TU> : LogixNode where TU : IEnumerable<T>
+    {
+        public readonly Impulse LoopStart;
+        public readonly Impulse LoopIteration;
+        public readonly Impulse LoopEnd;
+        public readonly Output<T> Iteration;
+
+        public readonly Input<TU> Collection;
+
+        //public readonly Input<bool> Reverse;
```

Comment:
?

Reply:
Actually on crack but when are we pushing this to main?

Reply:
probably now then amend fixes later